src/cloudflare/workers.ts

```diff
@@ -99,7 +99,10 @@ export const env = new Proxy(
     ): PropertyDescriptor | undefined {
       const inner = innerEnv.getCurrentEnv();
       if (inner) {
-        return Reflect.getOwnPropertyDescriptor(inner, prop);
+        const desc = Reflect.getOwnPropertyDescriptor(inner, prop);
+        if (desc) {
+          return { ...desc, enumerable: true };
```

Comment:
Odd, why are these properties not already enumerable? Where did they get marked non-enumerable?